cms/envs/test_docker.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'd prefer this live in a separate PR, or at least a separate commit. It seems unrelated (unless I'm missing something).